Formula/gst-plugins-good.rb

```diff
@@ -53,6 +53,18 @@ class GstPluginsGood < Formula
 
   depends_on "libogg" if build.with? "flac"
 
+  # This plugin causes hangs on Snow Leopard (and possibly other versions?)
+  # Upstream says it hasn't "been actively tested in a long time";
+  # successor is glimagesink (in gst-plugins-bad).
+  # https://bugzilla.gnome.org/show_bug.cgi?id=756918
+  # Also, it fails to build on Sierra.
+  # https://bugzilla.gnome.org/show_bug.cgi?id=770526
+  if MacOS.version == :snow_leopard or MacOS.version == :sierra
```

Comment:
Eh. I'd rather we not bother with the option until we know this is fixed upstream, since that's essentially an invite to attempt to build broken software.

Let's stick to the below `args` block & just tweak that to:
```ruby
if MacOS.version == :snow_leopard || MacOS.version == :sierra
  args << "--disable-osx_video"
end
```